scripts/release/proposal.js

```diff
@@ -50,40 +50,39 @@ try {
   // Make sure the release branch is up to date to prepare for new proposal.
   // The main branch is not automatically pulled to avoid inconsistencies between
   // release lines if new commits are added to it during a release.
-  run(`git checkout v${releaseLine}.x`)
-  run('git pull --ff-only')
+  run(`git checkout --quiet v${releaseLine}.x`)
+  run('git pull --quiet --ff-only')
 
   pass(`v${releaseLine}.x`)
 
   const diffCmd = [
     'branch-diff',
     '--user DataDog',
     '--repo dd-trace-js',
-  `--exclude-label=semver-major,dont-land-on-v${releaseLine}.x`
+    `--exclude-label=semver-major,dont-land-on-v${releaseLine}.x`
   ].join(' ')
 
   start('Determine version increment')
 
-  const lastVersion = require('../../package.json').version
-  const [, lastMinor, lastPatch] = lastVersion.split('.').map(Number)
+  const { DD_MAJOR, DD_MINOR, DD_PATCH } = require('../../version')
   const lineDiff = capture(`${diffCmd} --markdown=true v${releaseLine}.x master`)
   const isMinor = flags.minor || (!flags.patch && lineDiff.includes('SEMVER-MINOR'))
   const newVersion = isMinor
-    ? `${releaseLine}.${lastMinor + 1}.0`
-    : `${releaseLine}.${lastMinor}.${lastPatch + 1}`
+    ? `${releaseLine}.${DD_MINOR + 1}.0`
+    : `${releaseLine}.${DD_MINOR}.${DD_PATCH + 1}`
   const notesDir = path.join(os.tmpdir(), 'release_notes')
   const notesFile = path.join(notesDir, `${newVersion}.md`)
 
-  pass(`${isMinor ? 'minor' : 'patch'} (${lastVersion} -> ${newVersion})`)
+  pass(`${isMinor ? 'minor' : 'patch'} (${DD_MAJOR}.${DD_MINOR}.${DD_PATCH} -> ${newVersion})`)
 
   start('Checkout release proposal branch')
 
   // Checkout new or existing branch.
-  run(`git checkout v${newVersion}-proposal || git checkout -b v${newVersion}-proposal`)
+  run(`git checkout --quiet v${newVersion}-proposal &> /dev/null || git checkout --quiet -b v${newVersion}-proposal`)
```

Comment:
Why is this still needed with the latest version?

Reply:
I'm pretty sure I don't need the `--quiet` flag (do you want me to remove it?), but if I don't redirect the STDERR from the first command it will be picked up by the new error output detection code and the script aborted.

Reply:
Ok then the change is fine. I just wanted to make sure that it won't actually output, as long as that's true then it's fine 👍